### PR TITLE
Build the User Interface. 

### PR DESCRIPTION
**Pull request by KelynPNjeri**
This PR contains the code to the initial `landing page`, `voice input page`, `write number page`. To test the code, you need to clone the application and run **index.html** so that you can naviggate through the UI. Additional feature are currently a **WIP**.

### CODE REVIEW

**Comment by martinoywa** [comment]
Task linked: [CU-apyazp Build the User Interface](https://app.clickup.com/t/apyazp)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Number Tutor</title>
+    <link rel="stylesheet" href="assets/css/style.css">
```
Comment (martinoywa) [inline index.html:7]: This CSS doesn't seem to have been applied to the index file.

**Review by martinoywa — CHANGES_REQUESTED** [review]
Most things seem to be okay, but the design pattern followed is not modular as per 
this design doc https://app.clickup.com/4522322/v/dc/4a0aj-113/4a0aj-29

**Review by Mindo-Joseph — DISMISSED** [review]
Soo I had not run the code from the server, 😨 , the CSS was not loading ⚠️ , structure the code to look like below
Screenshot from 2020-12-09 14-19-15

**Comment by martinoywa** [comment]
Yeah, but I believe it's being worked on @KelynPNjeri?

**Comment by Mindo-Joseph** [comment]
Yeah but I also did it for the tests, will make a PR soon 😎

**Comment by martinoywa** [comment]
Perfect!

**Review by martinoywa — APPROVED** [review]

**Pull request merged**